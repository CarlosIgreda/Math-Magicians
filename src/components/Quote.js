import { Link } from 'react-router-dom';
import QuoteContainer from './QuoteContainer';

function Quote() {
  return (
    <div>
      <nav className="header">
        <h1 className="title">📐 Math Magicians📏</h1>
        <ul className="menu">
          <li className="menu-item">
            <Link to="/Math-Magicians">Home</Link>
          </li>
          <li className="menu-item">
            <Link to="/Math-Magicians/calculator">Calculator</Link>
          </li>
          <li className="menu-item">
            <Link to="/Math-Magicians/quote">Quote</Link>
          </li>
        </ul>
      </nav>
      <QuoteContainer />
    </div>
  );
}

export default Quote;
