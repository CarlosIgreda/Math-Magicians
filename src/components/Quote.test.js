import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Quote from './Quote';

describe('Quote', () => {
  it('should render correctly', () => {
    const { container } = render(
      <MemoryRouter>
        <Quote />
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
