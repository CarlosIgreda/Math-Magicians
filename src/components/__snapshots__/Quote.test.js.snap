// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote should render correctly 1`] = `
<div>
  <div>
    <nav
      class="header"
    >
      <h1
        class="title"
      >
        📐 Math Magicians📏
      </h1>
      <ul
        class="menu"
      >
        <li
          class="menu-item"
        >
          <a
            href="/Math-Magicians"
          >
            Home
          </a>
        </li>
        <li
          class="menu-item"
        >
          <a
            href="/Math-Magicians/calculator"
          >
            Calculator
          </a>
        </li>
        <li
          class="menu-item"
        >
          <a
            href="/Math-Magicians/quote"
          >
            Quote
          </a>
        </li>
      </ul>
    </nav>
    <p
      class="loading-text"
    >
      Loading...
    </p>
  </div>
</div>
`;
